doc/docportal/use_scummvm/taskbar_integration.rst

```diff
@@ -0,0 +1,46 @@
+===========================
+Taskbar integration
+===========================
+
+The taskbar integration has several features:
+
+- Adds an overlay icon in the taskbar/dock when running a game (Windows and macOS)
```

Comment:
...or Ubuntu

Reply:
Hmm ok - @criezy suggested it only works on Windows/macOS, which is why it's in there like that. I actually can't get it working on Ubuntu

Reply:
Indeed the overlay icon is only implemented on Windows and macOS. The taskbar integration on Linux is limited to the progress and count overlays used for the mass add feature.